apps/service_define/assets/src/component/presentation/data-definition/DataDefinition.stories.tsx

```diff
@@ -0,0 +1,197 @@
+import {storiesOf} from "@storybook/react"
+import React from "react"
+import {DataDefinition} from "./DataDefinition"
+import {DataDefinitionView} from "../../../model/view/DataDefinitionView"
+import {PrimitiveArgumentType} from "../../../model/view/ModuleFunctionArgsView"
+
+const simpleProps: DataDefinitionView = {
+    dataset_id: "houses",
+    subset_id: "default",
+    extract: {
+        source: {
+            struct_module_name: "Elixir.Extractor",
+            args: [
+                {
+                    key: "steps",
+                    type: ["list", PrimitiveArgumentType.module],
+                    value: [
+                        {
+                            struct_module_name: "Extract.Http.Get",
+                            args: [
+                                { key: "url", type: PrimitiveArgumentType.string, value: "http://www.source.com/the-data.csv"}
```

Comment:
Can we use example.com or something with .example as the tld just to be good denizens?

Reply:
The top-level domain example is explicitly intended to be used in documentation or other technical writing, when domain names are presented as examples in usage or presentation of concepts of the Domain Name System or the Internet.

Reply:
I almost suggested this while pairing and then thought "no, that's a little too pedantic". I'll put it in for you though Jessie.